### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by weiiwang01**
Applicable spec: <link>

### Overview

Update the README of the penpot charm based on the [new template for IS charms](https://github.com/canonical/is-charms-template-repo/blob/main/README.md).

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,76 @@
...
--->
 
-## Project and community
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+for deploying and managing the [penpot](https://penpot.app) open-source
+design tool for design and code collaboration in your systems.
+
+This charm simplifies the configuration and maintenance of penpot across a 
+range of environments, allows designers can create stunning designs, interactive prototypes, 
+design systems at scale, while developers enjoy ready-to-use code and make their workflow easy and fast.
```
Comment (erinecon) [inline README.md:14]: I would rephrase this to say:
```
This charm simplifies the configuration and maintenance of penpot across a 
range of environments, allowing designers to create stunning designs and interactive prototypes, 
design systems at scale, and make their workflow easy and fast with ready-to-use code.
```

File: README.md [hunk 2]
```diff
@@ -1,48 +1,76 @@
...
+First, we need to modify the `/etc/hosts` file to point the `penpot.local` domain to the IP address of the virtual machine.  
+After that, we can access the penpot instance in the browser using the address `https://penpot.local`.  
+Note that `https` is required for penpot to function. You may need to bypass the certificate security warning in the browser, as we are using a self-signed certificate.  
+
+Inside the virtual machine, run the following command to create a penpot account, and use the returned credentials to log in with this account.
+
+```
+juju run penpot/0 create-profile --string-args email=example@example.com fullname="John Doe"
+```
+
```
Comment (erinecon) [inline README.md:67]: Any there any integrations that are worth mentioning (in a separate section `## Integrations`, after the `## Get started` section)?

For instance, are the dependencies from the previous section required for penpot to work?

File: README.md (hunk 1 above)
Comment (weiiwang01) [reply README.md]: Updated, thanks!

File: README.md (hunk 2 above)
Comment (erinecon) [reply README.md:67]: As we discussed in our 1:1, I think you can mention the mandatory integrations in the `Integrations` section and then link to the Charmhub documentation to let the reader know about the full list of integrations

Comment (weiiwang01) [reply README.md:67]: Yes, updated with descriptions for all integrations. Thanks!

**Review by erinecon — COMMENTED** [review]
Thanks for updating the README -- I think it looks great! 

I approve of the changes 👍 not giving an official approval due to the changes in `.trivyignore`

**Review by amandahla — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,48 +1,105 @@
...
-<!--
-For DevOps or SRE teams this charm will make operating Penpot simple and
-straightforward through Juju's clean interface. It will allow easy deployment
-into multiple environments for testing of changes, and supports scaling out for
-enterprise deployments.
--->
 
-## Project and community
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+for deploying and managing the [penpot](https://penpot.app) open-source
```
Comment (varshigupta12) [inline README.md:9]: Nit: Capitalize Penpot

File: README.md [hunk 4]
```diff
@@ -1,48 +1,105 @@
...
+design tool for design and code collaboration in your systems.
+
+This charm simplifies the configuration and maintenance of penpot across a
```
Comment (varshigupta12) [inline README.md:12]: Same here, wherever the application name is used. For charm- you can keep it in lowercase

File: README.md (hunk 3 above)
Comment (varshigupta12) [reply README.md:9]: Nit: for deploying and managing [Penpot](https://penpot.app) "which is a web-based" open-source design tool for "prototyping, UI design,  and code collaboration"

File: README.md [hunk 5]
```diff
@@ -1,48 +1,105 @@
...
+```
+juju deploy penpot --channel latest/edge
+juju integrate penpot postgresql-k8s
+juju integrate penpot redis-k8s
+juju integrate penpot s3-integrator
+juju integrate penpot nginx-ingress-integrator
+```
+
+### Basic operations
+When the penpot charm has completed deployment and installation, you can access penpot from a browser.
```
Comment (varshigupta12) [inline README.md:57]: Nit: you can access "Penpot"

File: README.md [hunk 6]
```diff
@@ -1,48 +1,105 @@
...
+juju integrate penpot postgresql-k8s
+juju integrate penpot redis-k8s
+juju integrate penpot s3-integrator
+juju integrate penpot nginx-ingress-integrator
+```
+
+### Basic operations
+When the penpot charm has completed deployment and installation, you can access penpot from a browser.  
+First, we need to modify the `/etc/hosts` file to point the `penpot.local` domain to the IP address of the virtual machine.  
+After that, we can access the penpot instance in the browser using the address `https://penpot.local`.
```
Comment (varshigupta12) [inline README.md:59]: Same, capitalize

File: README.md (hunk 3 above)
Comment (weiiwang01) [reply README.md]: Capitalized all reference to Penpot, thanks!

**Review by varshigupta12 — APPROVED** [review]

**Review by yanksyoon — APPROVED** [review]
